src/main.rs

```diff
@@ -279,7 +277,14 @@ fn main() -> ExitCode {
 
 fn try_main(args: Cli, enable_styles: bool) -> Result<ExitCode> {
     // Look up config file, if none is found fall back to default config.
-    let config = Config::load(enable_styles).context("Could not load config")?;
+    debug!("Loading config");
+    let config = if args.config_path.is_some() && !args.seed_config {
+        Config::load(args.config_path.as_deref().unwrap(), enable_styles)
+            .context("Could not load config from given path")?
+    } else {
+        Config::load_default_path(enable_styles)
+            .context("Could not load config from default path")?
+    };
```

Comment:
How do you feel about the following? It avoids the unwrap, which makes it slightly easier in my opinion:
```suggestion
    let config = match &args.config_path {
        Some(path) if !args.seed_config => {
            Config::load(path, enable_styles).context("Could not load config from given path")?
        }
        _ => Config::load_default_path(enable_styles)
            .context("Could not load config from default path")?,
    };
```